test/lib.similar.js

```diff
@@ -8,4 +8,8 @@ describe('Similar method', () => {
     return store.similar({id: '553834731'})
     .then((apps) => apps.map(assertValidApp));
   });
+  it('should fetch a valid application list in fr country', () => {
```

Comment:
I've tried using 
```
assert(apps[0]['url'].slice(0, 27) === 'https://itunes.apple.com/fr', 'should return french app');
```
But it retrieves also apps from the us's store, so the test was failing.

On my side i checked that the result was not-null and different from the us's result. So i guess this is working.

But i did not include this test because i feel it is too weak to be part of a test-suite.